src/test/java/org/sagebionetworks/bridge/exporter/integration/ExportTest.java

```diff
@@ -0,0 +1,813 @@
+
+        // ensure schemas exist, so we have something to upload against
+        UploadSchemasApi uploadSchemasApi = developer.getClient(UploadSchemasApi.class);
+
+        UploadSchema legacySurveySchema = null;
+        try {
+            legacySurveySchema = uploadSchemasApi.getMostRecentUploadSchema("legacy-survey").execute().body();
+        } catch (EntityNotFoundException ex) {
+            // no-op
+        }
+        if (legacySurveySchema == null) {
+            UploadFieldDefinition def1 = new UploadFieldDefinition();
+            def1.setName("AAA");
+            def1.setType(UploadFieldType.SINGLE_CHOICE);
+
+            UploadFieldDefinition def2 = new UploadFieldDefinition();
+            def2.setName("BBB");
+            def2.setAllowOtherChoices(Boolean.FALSE);
+            def2.setType(UploadFieldType.MULTI_CHOICE);
+            def2.setMultiChoiceAnswerList(Lists.newArrayList("fencing", "football", "running", "swimming", "3"));
+
+            legacySurveySchema = new UploadSchema();
+            legacySurveySchema.setSchemaId("legacy-survey");
+            legacySurveySchema.setRevision(1L);
+            legacySurveySchema.setName("Legacy (RK/AC) Survey");
+            legacySurveySchema.setSchemaType(UploadSchemaType.IOS_SURVEY);
+            legacySurveySchema.setFieldDefinitions(Lists.newArrayList(def1,def2));
+            uploadSchemasApi.createUploadSchema(legacySurveySchema).execute();
+        }
+
+        ddbSynapseMetaTables.putItem(new Item().withPrimaryKey("tableName", studyId + "-appVersion"));
+        ddbSynapseMetaTables.putItem(new Item().withPrimaryKey("tableName", studyId + "-status"));
+        ddbSynapseTables.putItem(new Item().withPrimaryKey("schemaKey", studyId + "-legacy-survey-v1"));
+
+        uploadId = testUpload("./legacy-survey-encrypted");
+
+        // then create test synapse project and team
+        createSynapseProjectAndTeam();
+    }
+
+    @After
+    public void after() throws Exception {
+        ddbSynapseMetaTables.deleteItem("tableName", studyId + "-appVersion");
+        ddbSynapseMetaTables.deleteItem("tableName", studyId + "-status");
+        ddbSynapseTables.deleteItem("schemaKey", studyId + "-legacy-survey-v1");
+
+        if (studyId != null) {
+            admin.getClient(StudiesApi.class).deleteStudy(studyId, true).execute();
+        }
+        if (project != null) {
+            synapseClient.deleteEntityById(project.getId());
+        }
+        if (team != null) {
+            synapseClient.deleteTeam(team.getId());
+        }
+
+        ddbExportTimeTable.deleteItem("studyId", studyId);
+
+        if (StringUtils.isNotBlank(s3FileName)) {
+            s3Client.deleteObject(recordIdOverrideBucket, s3FileName);
+        }
+
+        admin.signOut();
+    }
+
+    @AfterClass
+    public static void deleteWorker() throws Exception {
+        if (worker != null) {
+            worker.signOutAndDeleteUser();
+        }
+    }
```

Comment:
it will respond account not found. even if it will not interrupt the test, i think its better to just delete the study and not delete each user afterwards. What do you think?